### PR TITLE
Fix remove filtering warning doc update

### PR DESCRIPTION
**Pull request by vincentdm05**
Update documentation to mention silent upper bound and max supported af level.

### CODE REVIEW

**Comment by vincentdm05** [comment]
👍

File: reference/imagetexture.md [hunk 1]
```diff
@@ -57,8 +57,11 @@ clamped in the *s* direction to lie within the [0.0,1.0] range. The `repeatT`
 field is analogous to the `repeatS` field.
 
 The `filtering` field defines the level of quality obtained from filtering
-techniques applied to the texture. This parameter ranges from 0 to 5, 0
-corresponding to no filtering, only performing simple nearest-neighbor pixel
+techniques applied to the texture. This parameter ranges from 0 to an integer
+increment relative to the maximum supported anisotropy level, which is usually 5.
```
Comment (omichel) [inline reference/imagetexture.md]: We should make it clear that 5 is the maximum value.

Comment (vincentdm05) [inline reference/imagetexture.md]: It's not always the maximum value, only in the case where it is related to the max supported level.

Comment (vincentdm05) [inline reference/imagetexture.md]: In general the max value is defined by the max supported level of the hardware.

**Comment by vincentdm05** [comment]
👍

File: reference/imagetexture.md [hunk 2]
```diff
@@ -57,8 +57,11 @@ clamped in the *s* direction to lie within the [0.0,1.0] range. The `repeatT`
 field is analogous to the `repeatS` field.
 
 The `filtering` field defines the level of quality obtained from filtering
-techniques applied to the texture. This parameter ranges from 0 to 5, 0
-corresponding to no filtering, only performing simple nearest-neighbor pixel
+techniques applied to the texture. This parameter ranges from 0 to an integer
+increment related to the maximum supported anisotropy level. This value usually is 5.
```
Comment (omichel) [inline reference/imagetexture.md]: For the sake of clarity I would use "value" instead of "increment" here.

**Pull request merged**